src/main/java/org/broadinstitute/hellbender/tools/ClipReads.java

```diff
@@ -460,7 +461,7 @@ private void accumulate(ReadClipperWithData clipper) {
         GATKRead clippedRead = clipper.clipRead(clippingRepresentation);
         if (outputBam != null) {
             outputBam.addRead(clippedRead);
-        } else {
+        } else if (out != null) {
```

Comment:
Is there a test that covers the case of `out == null`, and proves that the tool doesn't blow up in that case?